subprojects/gradle/build-verdict/src/main/kotlin/com/avito/android/build_verdict/BuildVerdictPluginExtension.kt

```diff
@@ -12,4 +14,32 @@ abstract class BuildVerdictPluginExtension(
 ) {
     val outputDir: Property<Directory> =
         objects.directoryProperty().convention(layout.projectDirectory.dir("outputs/build-verdict"))
+
+    val taskVerdictProviders = objects.listProperty<UserDefinedTaskVerdictProducer>()
```

Comment:
can you hide it with `internal`?
as far as i understand client supposed to use functions below?